airgun/views/computeprofile.py

```diff
@@ -0,0 +1,26 @@
+from widgetastic.widget import Text, TextInput
```

Comment:
put imports alphabetically

Reply:
you always put widgetastic.widget at the top, so I just forget the empty line, if I am right.
@oshtaier 
And what is the reason these lines and navmazing in entities are always on the top and also are entered from others ? It's like library we used but we don't implent them and modules which we implement after that ?

Reply:
probably reason for that is a common one:
first goes simple `imports` without `from`
then third party tools like widgetastic or nailgun
then our internal library modules